oshi-core/src/main/java/oshi/hardware/common/AbstractCentralProcessor.java

```diff
@@ -98,47 +98,56 @@ public AbstractCentralProcessor() {
 
     @Override
     public String getVendor() {
-        return cpuid.get().cpuVendor;
+        return getProcessorIdentifier().getVendor();
```

Comment:
These are all good, but don't forget to add a few lines above, a `getProcesserIdentifier()` implementation that returns `cpuid.get()`.